src/components/NurseryWithdrawals/PlantingProgressList.tsx

```diff
@@ -71,31 +76,85 @@ const columnsWithZones = (): TableColumnType[] => [
 export type PlantingProgressListProps = {
   search: string;
   plantingCompleted?: boolean;
+  reloadTracking: () => void;
 };
 
-export default function PlantingProgressList({ search, plantingCompleted }: PlantingProgressListProps): JSX.Element {
+export default function PlantingProgressList({
+  search,
+  plantingCompleted,
+  reloadTracking,
+}: PlantingProgressListProps): JSX.Element {
   const [hasZones, setHasZones] = useState<boolean | undefined>();
   const classes = useStyles();
   const data = useAppSelector((state: any) => searchPlantingProgress(state, search.trim(), plantingCompleted));
+  const [selectedRows, setSelectedRows] = useState<any[]>([]);
+  const dispatch = useAppDispatch();
+  const [requestId, setRequestId] = useState<string>('');
+  const updatePlantingResult = useAppSelector((state) => selectUpdatePlantingsCompleted(state, requestId));
 
   useEffect(() => {
     if (data && hasZones === undefined) {
       setHasZones(data.some((d) => d.subzoneName));
     }
   }, [data, hasZones]);
 
+  useEffect(() => {
+    if (updatePlantingResult?.status === 'success') {
+      reloadTracking();
+    }
+  }, [updatePlantingResult, reloadTracking]);
+
   if (!data || hasZones === undefined) {
     return <CircularProgress sx={{ margin: 'auto' }} />;
   }
 
+  const setPlantingComplete = (complete: boolean) => {
```

Comment:
rename to `setPlantingCompleted` just for consistency with other API and the property names?